### ISSUE TITLE
Add a11y: prefix to predefined

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#699]
If the a11y:certifiedBy property is retained in the Accessibility specification, it should be predefined to avoid declaring it every time.

No action should be taken on this until issue #698 is resolved, though.

**Comment by mattgarrish** [comment]
Prefix added in commit https://github.com/IDPF/epub-revision/commit/fe83be64bd6758173338f3838bb6eb0dad5e08a4
